imgui_widgets.cpp

```diff
@@ -7305,7 +7305,8 @@ static ImU32   ImGui::TabBarCalcTabID(ImGuiTabBar* tab_bar, const char* label, I
 {
     if (docked_window != NULL)
     {
-        IM_ASSERT(tab_bar->Flags & ImGuiTabBarFlags_DockNode);
+        IM_UNUSED(tab_bar);
+        IM_ASSERT(tab_bar->Flags & ImGuiTabBarFlags_DockNode);        
         ImGuiID id = ImHashStr(label);
```

Comment:
As a minor detail (probably more important for other PR) try to configure your IDE to trim trailing spaces.

I do know that VS2019 doesn't honor editor-config exactly like specs requires it (only trim trailing spaces when pressing Enter) but shouldn't be affected here.